docs/api/iteration_schemes.rst

```diff
@@ -0,0 +1,91 @@
+Iteration schemes
+=================
+
+IterationScheme
```

Comment:
In this file, you're making a call to `automodule` multiple times on the same module (by the way, it should be `fuel.schemes`, not `fuel.datasets.base`). Instead, the following should do the trick:

```
Iteration schemes
=================

.. automodule:: fuel.schemes
    :members:
    :undoc-members:
    :show-inheritance:
```